src/reactviews/index.css

```diff
@@ -43,6 +43,12 @@ button[role="combobox"] .placeholder {
     color: var(--vscode-input-placeholderForeground) !important;
 }
 
+.fui-Option:hover,
```

Comment:
I think the correct fix ought to be in `theme.ts`, rather than here.  Fixing the CSS directly rather than the Fluent theme means that other components that are still using the wrong theme variable remain dark.

Reply:
Thank you, and this suggestion is working better too.